### PR TITLE
Create primary_key_sql.md

### PR DESCRIPTION
**Pull request by verma-shivani**

### CODE REVIEW

**Comment by natemago** [comment]
@verma-shivani please use the [template](https://github.com/flashreads/blogs/blob/master/template.md) to set the meta-data for this article.

**Comment by verma-shivani** [comment]
@natemago I have made the required changes regarding the template . Please review this PR. 
And Sorry for the inconvinience.

File: databases/003-primary_key_sql.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+title: PRIMARY KEY IN SQL
+tags: Primary_key,implementation,Composite_key
+author: Shivani Verma
+meta-description: What is Primary Key?How it is implemented?How it is deleted and constraints related to it.
+---
+
+
+
+
+PRIMARY-KEY - SQL
```
Comment (ZoranPandovski) [inline databases/003-primary_key_sql.md:12]: Add # to make this as header

File: databases/003-primary_key_sql.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+A table can have only one primary key, which may consist of single or multiple fields. 
+When multiple fields are used as a primary key, they are called a composite key.
+
+If a table has a primary key defined on any field(s), then you cannot have two records having the same value of that field(s).
+
+
+To define PRIMARY KEY in sql:-
+
+      CASE 1:-when making the table
```
Comment (ZoranPandovski) [inline databases/003-primary_key_sql.md:25]: Please, format syntax with ```sql

Comment (verma-shivani) [reply databases/003-primary_key_sql.md]: okay

**Comment by verma-shivani** [comment]
made the required changes

**Review by ZoranPandovski — APPROVED** [review]

**Pull request merged**